telephoneNumberValidator/script.js

```diff
@@ -0,0 +1,40 @@
+const form = document.getElementById('my-form');
+let userInput = document.getElementById('user-input');
+let result = document.getElementById('is-palindrome');
+
+const TIMEOUT = 3000;
+
+function telephoneCheck(str) {
+    let regex = /^1{0,1}\s*(\(\d{3}\)|\d{3})(\s*|-)\d{3}(\s*|-)\d{4}$/;
+
+    if (regex.test(str)) return true;
+    
+    return false;
+}
+
+function timer() {
+    return setTimeout(() => {
+        result.textContent = '';
+        result.style.color = 'black';
```

Comment:
it's better to add all styles through classes. You can e.g. define a class `.black-text` in CSS and then add that class here

Reply:
> it's better to add all styles through classes. You can e.g. define a class `.black-text` in CSS and then add that class here

maybe i can remove this line with black text style since we only need green or red style?

Reply:
fixed